### ISSUE TITLE
RoBERTa on SuperGLUE's Broadcoverage Diagnostics task.

### ISSUE THREAD

**Issue by dirkgr** [allenai/allennlp#4992]
Broadcoverage Diagnostics is one of the tasks of the [SuperGLUE](https://super.gluebenchmark.com) benchmark. The task is to re-trace the steps of Facebook's RoBERTa paper (https://arxiv.org/pdf/1907.11692.pdf) and build an AllenNLP config that reads the Broadcoverage Diagnostics data and fine-tunes a model on it. We expect scores in the range of their entry on the [SuperGLUE leaderboard](https://super.gluebenchmark.com/leaderboard).

If you want, you can use the TransformerQA ([reader](https://github.com/allenai/allennlp-models/blob/main/allennlp_models/rc/dataset_readers/transformer_squad.py) and [model](https://github.com/allenai/allennlp-models/blob/main/allennlp_models/rc/models/transformer_qa.py)) code as a starting point.

**Comment by manishkumar771** [comment]
Hi.....Can i work on this task?

**Comment by dirkgr** [comment]
Absolutely! This is a binary classification task, so you'll need a dataset reader and a model. The model will probably be quite similar to the [multiple choice models](https://github.com/allenai/allennlp-models/tree/main/allennlp_models/mc) we already have.

Also, I don't believe we have the Matthew's Correlation metric, but it should not be hard to implement.

**Comment by dirkgr** [comment]
@wangyuxinwhy put together a similar model at https://github.com/wangyuxinwhy/SuperGLUE. In [his model config](https://github.com/wangyuxinwhy/SuperGLUE/blob/master/BoolQ/training_config/boolq.jsonnet), you can see he just used the existing `basic_classifier` and didn't write his own model at all. You can do the same thing and save a bunch of work!

**Comment by manishkumar771** [comment]
thanks for helping. I will look into it.

> @wangyuxinwhy put together a similar model at https://github.com/wangyuxinwhy/SuperGLUE. In [his model config](https://github.com/wangyuxinwhy/SuperGLUE/blob/master/BoolQ/training_config/boolq.jsonnet), you can see he just used the existing `basic_classifier` and didn't write his own model at all. You can do the same thing and save a bunch of work!
